server/service/hosts.go

```diff
@@ -1565,6 +1565,10 @@ func (svc *Service) HostEncryptionKey(ctx context.Context, id uint) (*fleet.Host
 		return nil, ctxerr.Wrap(ctx, newNotFoundError(), "getting host encryption key")
 	}
 
+	// TODO(mna): I don't think this is part of #13954, but this will need to be
+	// updated to support Windows encryption keys to use the WSTEP certificate
+	// instead of Apple SCEP.
```

Comment:
Anyone knows if this is covered by another ticket? Haven't been able to find it by a quick look at the story. Happy to change it directly in this PR if it's not covered (but needed).

Reply:
great point, I haven't seen a ticket for this.

Reply:
https://github.com/fleetdm/fleet/issues/14007